src/components/common/CommonFunctions.js

```diff
@@ -0,0 +1,4 @@
+export default function logoutClicked(setUserLoggedIn) {
```

Comment:
Looks like this function is logging user our. Should it be named like so (i.e logOut)? In general, functions performing authorization work should be in a file named something related to user auth like `authentication.js`. I propose we create a `/service` directory under the the `/src` directory. What do you think?

Reply:
This file is meant for generic functions used multiple places so I thought common was the best folder.  At the moment it is just the logout function but could hold other ones too. 

I think once we have more functions used by multiple files we can decide how to group them better but for now to me it seems easier to just have one file with a bunch of exported functions. 

CountryID issue is that the country IDs for a bunch of countries have seemingly changed over the past month. Germany has been saved in the past as a different number than it is being saved now. The saving is based on a geocoder result produced by react-map-gl-geocoder which should be using OpenStreetMaps. In order to color in the country map, the ID of the country needs to match a pre-determined ID value in the CountryInfo file (which was based on the "old" ID #'s). I put a temporary fix in here with an OR statement that allows country name (which hasn't changed) to match and result in a filled in country but long term this will need to be investigated and fixed in a new PR. It is as if the geocoder results have changed but I'm guessing that is not the case and I am just missing something.